components/EntryContentBlock/EntryContentBlock.style.js

```diff
@@ -1,7 +1,7 @@
 import { styled } from "styled-components";
 
 export const EntryContent = styled.div`
-  margin-top: 40px;
+  margin: 20px 0 20px 0;
```

Comment:
just a quick suggestion, but because you are using the same margin for top/bottom and left/right you could also write the code like this: `margin: 20px 0;`
[I looked that up](https://developer.mozilla.org/en-US/docs/Web/CSS/margin#syntax) about half an hour ago myself so that's why I noticed it. Your code also works though, so no need to actually change something.

Reply:
thanks for the tipp! :)